en/docs/install-and-setup/deploying-wso2-api-manager/production-deployment-guidelines.md

```diff
@@ -62,10 +65,13 @@ Prior to installing any WSO2 Carbon-based product, it is necessary to have the a
 
 ### Environment compatibility
 
--   By default, WSO2 products are installed with **OpenJDK** , which allows you to run the product as soon as it is installed.
+-   If you are using the product installer to install the product, by default, WSO2 API-M is installed with OpenJDK, 
+which allows you to run the product as soon as it is installed. If you are using the product's binary distribution to
+ install the product (instead of the product installer), install JDK. Make sure your JDK version is compatible with 
+ the WSO2 product.
 
-    !!! tip
-        To use a different JDK, point the **`JAVA_HOME           `** environment variable to the new JDK. Make sure your JDK version is [compatible with the WSO2 product](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems+and+JDKs) .
+!!! tip
+    To use a different JDK, point the **`JAVA_HOME`** environment variable to the new JDK. Make sure your JDK version is [compatible with the WSO2 product](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems+and+JDKs) .
```

Comment:
shall we remove the extra space before the full stop?

Reply:
Fixed with dd3957d